Sources/Sentry/include/HybridPublic/PrivateSentrySDKOnly.h

```diff
@@ -163,7 +163,8 @@ typedef void (^SentryOnAppStartMeasurementAvailable)(
 
 #endif // SENTRY_UIKIT_AVAILABLE
 
-#if SENTRY_HAS_UIKIT && !TARGET_OS_VISION
+#if SENTRY_TARGET_REPLAY_SUPPORTED
++ (SentrySessionReplayIntegration *__nullable)getReplayIntegration;
```

Comment:
m: Why do you need the integration? The `configureSessionReplay` is not enough?

Reply:
we probably don't, removing from the header.
```suggestion
```